packages/jaeger-ui/src/components/TraceDiff/TraceDiff.tsx

```diff
@@ -46,7 +46,7 @@ type TDispatchProps = {
 
 type TOwnProps = {
   history: RouterHistory;
-  match: match<TDiffRouteParams>;
+  params: TDiffRouteParams;
```

Comment:
Does this change have any impart on the shape of the URL? IIUC the state is included in the URL.

Reply:
No. Absolutely not. 

Previous URL structure: 
```
http://localhost:5173/trace/4e9e0a8990106fdd...196659de1e174bd4?cohort=4e9e0a8990106fdd&cohort=196659de1e174bd4
```

Current:
```
http://localhost:5173/trace/51c935040996787d...0f7faf874d731c2c?cohort=51c935040996787d&cohort=0f7faf874d731c2c
```

It's just the place from where we use the extracted URL parameters.